File: sorting/selection_sort.py
arr = [1,5,2,13,4, 18, 3, 0, 100]

def selection_sort(arr):
    length = len(arr)
    for i in range(0, length):
        minimum = i
        for j in range(i, length):
            if arr[i] > arr[j]:
                minimum = j
                temp = arr[i]
                arr[i] = arr[minimum]
                arr[minimum] = temp
    return arr

if __name__ == "__main__":
    print(selection_sort(arr=arr))

File: sorting/selectionSort.js
const selectionSort = (arr => {
    const length = arr.length;
    for (let i = 0; i < length; i++){
        let minimum = i;
        for (let j = i; j < length; j++){
            if (arr[i] > arr[j]){
                minimum = j;
                temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
    }
    return arr;
})

const arr = [1,5,2,13,4,18,3,0,100];
console.log(selectionSort(arr));